app/gitcoinbot/tests.py

```diff
@@ -81,21 +87,73 @@ def test_parse_tipee_username(self):
+        tip_link = f'{settings.BASE_URL}tip/?amount=3.3&tokenName=ETH&username=@user&source={issue_url}'
         target_text = f"To complete the tip, please [visit this link]({tip_link}).\n " \
             "PS Make sure you're logged into Metamask!"
         text = new_tip_text("test_owner", "gitcoin", "1234", "@user 3.3 ETH")
         self.assertEqual(text, target_text)
 
-    def test_claim_bounty_text(self):
-        """Test Gitcoinbot can respond with link to claim bounty."""
-        issue_url = "https://github.com/test_owner/gitcoin/issues/1234"
-        claim_link = f"{settings.BASE_URL}funding/details/?url={issue_url}"
-        target_text = f"To finish claiming this bounty please [visit this link]({claim_link})"
-        text = claim_bounty_text("test_owner", "gitcoin", "1234")
+    def test_submit_work_text(self):
+        """Test Gitcoinbot can respond with link to submit your work."""
+        submit_link = f"{settings.BASE_URL}issue/test_owner/gitcoin/1234"
+        target_text = f"To finish claiming this bounty please [visit this link]({submit_link})"
+        text = submit_work_text("test_owner", "gitcoin", "1234")
+        self.assertEqual(text, target_text)
+
+    def test_start_work_text(self):
+        start_work_link = f"{settings.BASE_URL}issue/test_owner/gitcoin/1234"
+        target_text = f"To show this bounty as started please [visit this link]({start_work_link})"
+        text = start_work_text("test_owner", "gitcoin", "1234")
         self.assertEqual(text, target_text)
 
     def test_confused_text(self):
         """Test Gitcoinbot can respond that it's confused."""
         self.assertEqual(confused_text(),
             'Sorry I did not understand that request. Please try again or use `@gitcoinbot help` to see supported commands.')
+
+    def test_submit_work_or_new_bounty_when_bounty_exists(self):
+        from dashboard.models import Bounty
+        from datetime import datetime
+        Bounty.objects.create(
+            title='foo',
+            value_in_token=3,
+            token_name='ETH',
+            web3_created=datetime(2008, 10, 31),
+            github_url='https://github.com/test_owner/gitcoin/issues/1234',
+            token_address='0x0',
+            issue_description='hello world',
+            bounty_owner_github_username='flintstone',
+            is_open=False,
+            accepted=True,
+            expires_date=datetime(2008, 11, 30),
+            idx_project_length=5,
+            project_length='Months',
+            bounty_type='Feature',
+            experience_level='Intermediate',
+            raw_data={},
+        )
+
+        submit_link = f"{settings.BASE_URL}issue/test_owner/gitcoin/1234"
+        target_text = f"To finish claiming this bounty please [visit this link]({submit_link})"
+        text = submit_work_or_new_bounty_text("test_owner", "gitcoin", "1234")
+        self.assertEqual(text, target_text)
+
```

Comment:
Can you remove one of the blank lines here?

Reply:
I removed the previous blank line :smiling_imp: